jaxrs/src/test/java/se/fortnox/reactivewizard/jaxrs/JaxRsResourceTest.java

```diff
@@ -746,6 +746,21 @@ public void shouldAcceptBeanParamInherited() {
         assertThat(get(service, "/test/acceptsBeanParamInherited?name=foo&age=3&items=1,2&inherited=YES").getOutp()).isEqualTo("\"foo - 3 2 - YES\"");
     }
 
+    @Test
+    public void shouldGive400ErrorForInvalidHexByteInQuery() {
+        assertBadRequest("/test/accepts?myarg=%A");
+        assertBadRequest("/test/accepts?%A");
+        assertBadRequest("/test/accepts?%=");
+        assertBadRequest("/test/accepts?%?");
+        assertBadRequest("/test/accepts?%AG");
+        assertBadRequest("/test/accepts?%A@");
+    }
+
+    private void assertBadRequest(String uri) {
+        MockHttpServerResponse mockHttpServerResponse = get(service, uri);
+        assertThat(mockHttpServerResponse.status()).isEqualTo(HttpResponseStatus.BAD_REQUEST);
```

Comment:
Use LoggingVerifier to assert the logging as well.

Reply:
:+1: